specificatie/openapi.yaml

```diff
@@ -300,7 +333,7 @@ components:
       type: object
       properties:
         _links:
-          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks'
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/schemas/HalPaginationLinksMetLast'
```

Comment:
@kad-tromps kunnen jullie bij paginering ook een last-link aanbieden? Of verlies je dan het voordeel van paginering omdat de query alsnog de hele lijst door moet?

Reply:
Vooralsnog staat deze nog onbeantwoord. Kan ook worden aangepast als de reactie van @kad-tromps binnen is.

Reply:
Ik ben niet enorm thuis in de technische specifieken van Oracle, zou ik even met onze database expert moeten bespreken. Mijn inschatting is dat ook als je een paging doet, je toch de query over de hele tabel moet uitvoeren, om te bepalen op welke selectie je moet pagen. Misschien is het mogelijk om bij deze selectie ook direct de totale grootte op te halen, waarmee de laatste pagina eenvouding is af te leiden. 

Dit is voor mij nog steeds een misschien, echter, dus het lijkt me slim om de last link als verbetering te beschouwen.

Reply:
OK, dan doe ik hem eerst zonder de last page.